recipes/copier-templates-extensions/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "copier-templates-extensions" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 48b3f5213ee913b931df40e4e9325b58b9c9a741f1052a7da1b247ed991f5150
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - pdm-pep517 <1.0.5
+  run:
+    - python >=3.7
+    - copier >=6
+
+test:
+  imports:
+    - copier_templates_extensions
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/copier-org/copier-templates-extensions/blob/master/pyproject.toml
```

Comment:
is this really supposed to point to the `pyproject.toml` file?

Reply:
oups - thanks Wolf